pups.txt

```diff
@@ -0,0 +1,122 @@
+# -------------------------------------[INFO]---------------------------------------
+#
+# Unwanted Program list
+# ------------------------------------[FILTERS]-------------------------------------
+0.0.0.0 www.totalav.com
+0.0.0.0 totalav.com
+0.0.0.0 api.totalav.com
+0.0.0.0 secure.totalav.com
+0.0.0.0 url.totalav.com
+0.0.0.0 support.totalav.com
+0.0.0.0 blog.totalav.com
+0.0.0.0 track.totalav.com
+0.0.0.0 ajax.totalav.com
+0.0.0.0 affiliate.totalav.com
+0.0.0.0 livechat.totalav.com
+0.0.0.0 advertisers.totalav.com
+0.0.0.0 affiliates.totalav.com
+0.0.0.0 my.totalav.com
+0.0.0.0 assets.totalav.com
+0.0.0.0 identity.totalav.com
+0.0.0.0 login.totalav.com
+0.0.0.0 download.totalav.com
+0.0.0.0 static.totalav.com
+0.0.0.0 adblock.totalav.com
+0.0.0.0 sso.totalav.com
+0.0.0.0 webshield.totalav.com
+0.0.0.0 resources.totalav.com
+0.0.0.0 signup.totalav.com
+0.0.0.0 link.totalav.com
+0.0.0.0 chat.totalav.com
+0.0.0.0 click.totalav.com
+0.0.0.0 stats.totalav.com
+0.0.0.0 search.totalav.com
+0.0.0.0 aff.totalav.com
+0.0.0.0 news.totalav.com
+0.0.0.0 blockpage.totalav.com
+0.0.0.0 ext.totalav.com
+0.0.0.0 smtpmail.totalav.com
+0.0.0.0 articles.totalav.com
+0.0.0.0 data.totalav.com
+0.0.0.0 pda.totalav.com
+0.0.0.0 firmy.totalav.com
+0.0.0.0 portal.totalav.com
+0.0.0.0 educa.totalav.com
+0.0.0.0 cp.totalav.com
+0.0.0.0 images.totalav.com
+0.0.0.0 p.totalav.com
+0.0.0.0 gallery.totalav.com
+0.0.0.0 webshop.totalav.com
+0.0.0.0 new.totalav.com
+0.0.0.0 sklep.totalav.com
+0.0.0.0 manitoba.totalav.com
+0.0.0.0 wiki.totalav.com
+0.0.0.0 pei.totalav.com
+0.0.0.0 dl.totalav.com
+0.0.0.0 bbs.totalav.com
+0.0.0.0 schools.totalav.com
+0.0.0.0 ts.totalav.com
+0.0.0.0 hosting.totalav.com
+0.0.0.0 test.totalav.com
+0.0.0.0 live.totalav.com
+0.0.0.0 eng.totalav.com
+0.0.0.0 forums.totalav.com
+0.0.0.0 lnx.totalav.com
+0.0.0.0 lib.totalav.com
+0.0.0.0 galeria.totalav.com
+0.0.0.0 cloud.totalav.com
+0.0.0.0 winzipregistryoptimizer.com
+0.0.0.0 www.winzipregistryoptimizer.com
+0.0.0.0 download.winzipregistryoptimizer.com
+0.0.0.0 store.winzipregistryoptimizer.com
+0.0.0.0 ww.winzipregistryoptimizer.com
+0.0.0.0 mail.winzipregistryoptimizer.com
+0.0.0.0 www.www.winzipregistryoptimizer.com
+0.0.0.0 winzip.com
+0.0.0.0 api.winzip.com
+0.0.0.0 installprotector-api.winzip.com
+0.0.0.0 www.winzip.com
+0.0.0.0 tray.winzip.com
+0.0.0.0 updaterv.winzip.com
+0.0.0.0 download.winzip.com
+0.0.0.0 store.winzip.com
+0.0.0.0 goto.winzip.com
+0.0.0.0 help.winzip.com
+0.0.0.0 courier.winzip.com
+0.0.0.0 support.winzip.com
+0.0.0.0 go2.winzip.com
+0.0.0.0 image.winzip.com
+0.0.0.0 update.winzip.com
+0.0.0.0 systemtools.winzip.com
+0.0.0.0 downloads.winzip.com
+0.0.0.0 shop.winzip.com
+0.0.0.0 activate.winzip.com
+0.0.0.0 subscribe.winzip.com
+0.0.0.0 install.winzip.com
+0.0.0.0 secure.winzip.com
+0.0.0.0 response.winzip.com
+0.0.0.0 download-origin.winzip.com
+0.0.0.0 donwload.winzip.com
+0.0.0.0 www6.winzip.com
+0.0.0.0 openvpn.winzip.com
```

Comment:
`winzip.com`?? as I recall it winzip is a rather widely used packing program for shitdows!

Reply:
@spirillen let me check.
I took these from my lists, so there may be a false positive

Reply:
> `https://safeweb.norton.com/reviews?url=winzip.com` 

Is actually spyware....  

```shell
drill -T safeweb.norton.com
net.    172800  IN      NS      m.gtld-servers.net.
trafficmanager.net.     172800  IN      NS      tm1.edgedns-tm.info.
trafficmanager.net.     172800  IN      NS      tm2.edgedns-tm.info.
trafficmanager.net.     172800  IN      NS      tm1.dns-tm.com.
trafficmanager.net.     172800  IN      NS      tm2.dns-tm.com.
shasta-web-prod.trafficmanager.net.     60      IN      A       52.224.14.113
```

`trafficmanager.net`: https://mypdns.org/my-privacy-dns/matrix/-/issues/2507

Reply:
for the winzip, five it a spin in your virtual install.

Reply:
> > `https://safeweb.norton.com/reviews?url=winzip.com`
> 
> Is actually spyware....
> 
> ```shell
> drill -T safeweb.norton.com
> net.    172800  IN      NS      m.gtld-servers.net.
> trafficmanager.net.     172800  IN      NS      tm1.edgedns-tm.info.
> trafficmanager.net.     172800  IN      NS      tm2.edgedns-tm.info.
> trafficmanager.net.     172800  IN      NS      tm1.dns-tm.com.
> trafficmanager.net.     172800  IN      NS      tm2.dns-tm.com.
> shasta-web-prod.trafficmanager.net.     60      IN      A       52.224.14.113
> ```
> 
> `trafficmanager.net`: https://mypdns.org/my-privacy-dns/matrix/-/issues/2507

@spirillen ?
safeweb.norton.com is owned by Norton - https://www.virustotal.com/gui/url/a72dfee23d9db915aeb3312a0518fcb84fc4eb2c09253145bf9b3a7c4dcadfbc/community

Reply:
@spirillen it is late here so I am logging off my computer and will handle this tomorrow

Reply:
norton's safeweb is using the spyware network `trafficmanager.net`, this network is used to pick up any data on you :shrug:

Reply:
@spirillen 
1. That does not mean that their ratings are horrible
2. I have other sources to verify it too
3. Just because a website uses trackers does not mean it is malware. I am pretty sure that would be blocked anyway by uBo

I am very busy this week, so I will not see replies for hours or days

Reply:
Did blame any of it :smirk: 

I know I wrote `Is actually spyware....` It wasn't ittendent to the safeweb project it self, more it was served over a spooky network.

Sorry if it wasn't clear